llvm/lib/Transforms/IPO/MergeFunctions.cpp

```diff
@@ -404,7 +430,7 @@ bool MergeFunctions::runOnModule(Module &M) {
     // If the hash value matches the previous value or the next one, we must
     // consider merging it. Otherwise it is dropped and never considered again.
     if ((I != S && std::prev(I)->first == I->first) ||
-        (std::next(I) != IE && std::next(I)->first == I->first) ) {
+        (std::next(I) != IE && std::next(I)->first == I->first)) {
```

Comment:
Nit: you should revert the whitespace fix-up on this line since it's not related to the patch anymore and makes it hard to see what actually changed. (Feel free to commit it separately / before / after as an NFC change if you want to clean it up (no need for review, IMO), as long as it's not squashed with this change.)

Reply:
The GitHub pull request interface requires "squashing", so if you don't have commit access to push this separately, it'd be better just to leave it out. Sorry for the churn!